packages/core/src/components/NestedForm/NestedForm.tsx

```diff
@@ -10,6 +10,7 @@ import { BoundComponent } from '../bind';
 import { OutsideFormError } from '../../errors';
 
 export interface NestedFormProps {
+    validatorTrigger?: string;
```

Comment:
How is this prop being used?

would you be able to write a unit test for this to assert that the desired behaviour is working correctly?

Reply:
I'm guessing this is helping  down the way in `Form.tsx=>getComponentValidatorTriggers=>getComponentProps(componentName)` to get the `validatorTrigger` props, without this `props.validatorTrigger` will always undefined